client/src/components/categorical/value/index.js

```diff
@@ -336,11 +427,11 @@ class CategoryValue extends React.Component {
       LABEL_MARGIN +
       (isUserAnno ? ANNO_MENU : 0);
 
-    const OCCUPANCY_WIDTH = 100;
+    const GRAPH_WIDTH = 100;
```

Comment:
For better or worse, we use `graph` for the main scatterplot. Alternatives include a more specific `CANVAS_WIDTH`, `CHART_WIDTH` or `MINI_HISTO_WIDTH`